master_mapping.py

```diff
@@ -148,6 +148,14 @@
         "description": "wheel state",
         "decoder": decode35,
     },
+    48879: {
+        "description": "MPU Motor States",
+        "decoder": decodeMotorStates,
```

Comment:
^echoing previous comment, should be called something more related to MPU states rather than dragging the motor into this